SetUp/autoDefine/autoDefine.py

```diff
@@ -277,16 +278,19 @@ def inputBuilder( directories, options, keep_going, save_intermed ):
             os.remove('def.input')
             os.remove('def.out')
 
-#        p=sp.Popen('cosmoprep < templateCosmo.input > cosmoprep.out',shell=True)
-#        p.wait()
-#
-#
-#        #Place setup files no longer needed in setup dir
-#        if not os.path.exists('setup'):
-#            os.makedirs('setup')
-#        for i in ['input.xyz','options','templateCosmo.input','cosmoprep.out','def.input','def.out']:
-#            try: os.rename(i,'setup/'+i)
-#            except OSError: pass
+        if cosmo:
```

Comment:
This is better than before, but I still don't want this in the code until cosmoprep.input is generated by the script